addOns/websocket/src/test/java/org/zaproxy/zap/extension/websocket/alerts/WebSocketAlertRaiserUnitTest.java

```diff
@@ -43,7 +44,9 @@ void shouldBuildAlert() {
         // Given
         WebSocketAlertThread webSocketAlertThread = mock(WebSocketAlertThread.class);
         WebSocketMessageDTO message = mock(WebSocketMessageDTO.class);
-        message.channel = mock(WebSocketChannelDTO.class);
+        given(message.getChannel()).willReturn(mock(WebSocketChannelDTO.class));
+        given(message.getChannel().getId()).willReturn(1);
+        message.setChannel(mock(WebSocketChannelDTO.class));
```

Comment:
Does not seem to be needed, already being mocked above.

Reply:
Was getting an NPE without the givens

Reply:
Right, but that's for the above then? (Which I missed that it returns an Integer not an int.)

Reply:
They had both been NPEing. Will test/tweak further